testpilot/frontend/static-src/app/main.js

```diff
@@ -40,6 +41,7 @@ app.extend({
   initialize() {
     app.webChannel = webChannel;
     app.me = new Me();
+    app.clipboard = new Clipboard('button');
```

Comment:
I'm not familiar w/ [**clipboard**](https://www.npmjs.com/package/clipboard) usage, but should we make this be a more specific selector? IIUC, this could potentially break if we added a second `<button>` element to the page.